src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/ProcessTask.java

```diff
@@ -148,7 +148,7 @@ public TaskResult call() {
         scope.addDimension(MetricsHelper.SHARD_ID_DIMENSION_NAME, shardInfo.getShardId());
         scope.addData(RECORDS_PROCESSED_METRIC, 0, StandardUnit.Count, MetricsLevel.SUMMARY);
         scope.addData(DATA_BYTES_PROCESSED_METRIC, 0, StandardUnit.Bytes, MetricsLevel.SUMMARY);
-
+        getRecordsCache.setMetricOperation(this.getClass().getSimpleName());
```

Comment:
I wouldn't use this. I would pass the value as a String "ProcessTask"

Reply:
what's the difference between hardcode "PorcessTask" here and hardcode it inside startScope?

Reply:
The difference is that of the class name changes or gets refactored, we want the metric operation to be the same. That is why hardcoding it would be better.